ivy/functional/frontends/numpy/ndarray/ndarray.py

```diff
@@ -388,6 +388,9 @@ def __imul__(self, value, /):
     def __itruediv__(self, value, /):
         return np_frontend.true_divide(self._ivy_array, value)
 
+    def __ifloordiv__(self, value, /):
+        return np_frontend.floor_divide(self.ivy_array, value, out=self)
```

Comment:
This should be `self._ivy_array`  in floor_divide(), and also remove the `out=self`.

Thanks!